user_journey/user_journey.model.lkml

```diff
@@ -0,0 +1,166 @@
+connection: "telemetry"
+
+include: "//looker-hub/user_journey/views/events_daily.view.lkml"
+include: "/user_journey/views/*/*.view.lkml"                # include all views in the views/ folder in this project
+include: "/user_journey/views/*.view.lkml"
+
+explore: funnel_analysis {
+  view_label: " User-Day Funnels"
+  join: days_of_use {
+    view_label: "Days of Use"
+    relationship: one_to_one
+    type: left_outer
+    sql_on: ${funnel_analysis.sample_id} = ${days_of_use.sample_id}
```

Comment:
This gives us either the most recent search_clients_last_seen, or the one that is 56 days in the future.